src/tests/test_claim_detector.py

```
@@ -0,0 +1,254 @@
+        )
+
+        # Test basic properties
+        assert claim.text == "Test claim"
+        assert claim.context == "Test context"
+        assert claim.check_worthiness == 0.7
+        assert claim.domain == "Science"
+        assert claim.confidence == 0.8
+        assert claim.entities == ["test", "claim"]
+
+        # Test methods
+        assert claim.is_checkworthy(threshold=0.5) == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if claim.is_checkworthy(threshold=0.5):` for truth checks

Replace with `claim.is_checkworthy(threshold=0.5)`

(E712)